examples/jsm/postprocessing/BloomPass.js

```diff
@@ -10,15 +10,15 @@ import { ConvolutionShader } from '../shaders/ConvolutionShader.js';
 
 class BloomPass extends Pass {
 
-	constructor( strength = 1, kernelSize = 25, sigma = 4, resolution = 256 ) {
+	constructor( strength = 1, kernelSize = 25, sigma = 4 ) {
 
 		super();
 
 		// render targets
 
-		this.renderTargetX = new WebGLRenderTarget( resolution, resolution );
+		this.renderTargetX = new WebGLRenderTarget( 1, 1 );
 		this.renderTargetX.texture.name = 'BloomPass.x';
-		this.renderTargetY = new WebGLRenderTarget( resolution, resolution );
+		this.renderTargetY = new WebGLRenderTarget( 1, 1 );
```

Comment:
I think that will cause confusion to users without an explanation.

How about adding a comment:
```js
this.renderTargetY = new WebGLRenderTarget( 1, 1 ); // will be resized later
```
Maybe there is a more-precise comment you could add.

Reply:
How about we add `1` as a default parameter for `width` and `height` in `WebGLRenderTraget`s ctor?

https://github.com/mrdoob/three.js/blob/03efa5a594bc44705a0dda94711a76c8e75f21ae/src/renderers/WebGLRenderTarget.js#L14

In this way we can just write `const renderTarget = new WebGLRenderTarget();` which simplifies the code even more. We also allow to other objects like geometries, materials and 3D objects to be created without arguments.

Reply:
Hehe. I suggested that too, but removed it for simplicity. That is fine.

I still think adding the comment would be helpful, though.

Reply:
Okay, then let me add the comments for now and change the ctor of `WebGLRenderTarget` with a different one.